fcrepo-http-commons/src/test/java/org/fcrepo/http/commons/session/TransactionProviderTest.java

```diff
@@ -34,14 +34,13 @@
 import java.net.URI;
 
 import static java.net.URI.create;
-import static org.fcrepo.http.commons.session.TransactionProvider.JMS_BASEURL_PROP;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.spy;
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
 /**
- * @author pwinckles
+ * @author pwinckles (I definitely did not write these)
```

Comment:
I'm not a huge fan of this as makes it seem like there is something wrong with the tests.

Based on the PR you added it with (https://github.com/fcrepo/fcrepo/pull/1661) you say you moved it here, but I can't see where you moved it from. If we can determine that, then we can identify the actual author, though now that you have altered it you'd be an author as well.

Reply:
Removed